Content.Client/_RMC14/Marines/MarineOverlay.cs

```diff
@@ -36,8 +37,9 @@ public MarineOverlay()
 
     protected override void Draw(in OverlayDrawArgs args)
     {
-        if (!_entity.HasComponent<MarineComponent>(_players.LocalEntity))
-            return;
+        if (!_entity.HasComponent<CMGhostMarineHudComponent>(_players.LocalEntity))
+            if (!_entity.HasComponent<MarineComponent>(_players.LocalEntity))
+                return;
```

Comment:
These two if statements can be merged 
```suggestion
        if (!_entity.HasComponent<CMGhostMarineHudComponent>(_players.LocalEntity) &&
            !_entity.HasComponent<MarineComponent>(_players.LocalEntity))
            {
                return;
            }
```